dist/knockout-interpolate.js

```diff
@@ -18,21 +18,42 @@
         return node.nodeType === 3 && node.nodeValue.indexOf("{{") > -1;
     }
 
-    ko.bindingProvider.instance = {
+    function hasInterpolationAttribute(node){
+        return node.attributes && node.attributes["data-koset"];
+    }
+
+    function getValueOfExpression(rawExpression, bindingContext, node){
+        var expression = rawExpression.replace("{{", "").replace("}}", "");
+        // take advantage of existing KO functionality to parse/evaluate binding expression
+        return ko.unwrap(defaultProvider.parseBindingsString("x:" + expression, bindingContext, node).x);
+    }
+
+    function processInterpolationAttribute(node, bindingContext){
+        var bindingValues = defaultProvider.parseBindingsString(node.attributes["data-koset"].value, bindingContext, node);
```

Comment:
Might want to consider doing `ko.unwrap` on values that you are using, if you want to support someone passing an observable without parens (obviously it would still not update if the observable changed).